README.md

```diff
@@ -204,10 +208,16 @@ ambient occlusion is disabled.
 sky texture that surrounds the scene. The texture is not used for
 baking.
 
+`ambientLight` has the following properties:
+
++ `strength` optional, a float that specifies how strong is the effect
+of ambient light, defaults to `0.05`. `0` is an equivalent of
+disabled ambient lighting.
+
 `ambientOcclusion` has the following properties:
 
-+ `factor` optional, a float that specifies how strong is the effect
-of ambient occlusion, defaults to `0.05`. `0` is an equivalent of
++ `intensity` optional, a float that specifies how strong is the effect
+of ambient occlusion, defaults to `0.5`. `0` is an equivalent of
```

Comment:
a float that specifies how strong the effect of ambient occlusion is (passive voice)

I'd drop the 0 is an equivalent...

Reply:
Good point, thanks